llm/payload.go

```diff
@@ -15,20 +15,32 @@ import (
 
 	"golang.org/x/sync/errgroup"
 
+	"github.com/ollama/ollama/envconfig"
 	"github.com/ollama/ollama/gpu"
 )
 
 var errPayloadMissing = errors.New("expected payloads not included in this build of ollama")
 
 func Init() error {
-	payloadsDir, err := gpu.PayloadsDir()
+	binGlob := "build/*/*/*/bin/*"
+	extract := true
+
+	// Short circuit if we intentionally have no payloads
+	files, err := fs.Glob(libEmbed, binGlob)
+	if err != nil || len(files) == 0 || (len(files) == 1 && strings.Contains(files[0], "NO_PAYLOAD")) {
```

Comment:
should instead be looking to see if the runner binaries are in the right location? (vs a hardcoded directory)